kbase-extension/static/kbase/css/landingPages.css

```diff
@@ -351,11 +351,6 @@ table {
   padding-left: 20px;
   padding-right: 20px;
   border: 1px solid #428bca;
-
-  /* FIXME: why does the font change here?! */
-
-  font-family: OxygenBold;
-  font-size: 14px;
```

Comment:
I can't even find anywhere that this style is used, so deleting this aberrant font declaration for `.active .btn-narr`.